src/permutation.jl

```diff
@@ -0,0 +1,65 @@
+export PermutationTest
+
+function ptstats(x,y,f)
+    xy = vcat(x,y)
+    rx = 1 : length(x)
```

Comment:
No spaces around `:` here and below. Use parentheses where needed to make the precedence rules more immediately visible.